controllers/webpay_plus.js

```diff
@@ -37,22 +37,42 @@ exports.create = asyncHandler(async function (request, response, next) {
 
 exports.commit = asyncHandler(async function (request, response, next) {
   let token = request.body.token_ws;
+  if (token){
```

Comment:
There is a case where the token is returned and the transaction has not completed. We have to check the flow.